wdqs/0.3.1/.travis/build-deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+#Oneline for full directory name see: https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+set -e
+docker build --pull "$DIR/../" -t wikibase/wdqs:0.3.1 -t wikibase/wdqs:latest
+
+if [ "$SHOULD_DOCKER_PUSH" = true ]; then
```

Comment:
Bash scripts should use `[[`, which also makes quoting unnecessary:

    if [[ $SHOULD_DOCKER_PUSH == true ]]; then

(Both `=` and `==` work, but `==` is the canonical form, and I find it more readable.)